learning/tour-of-beam/frontend/assets/translations/en.yaml

```diff
@@ -15,15 +15,10 @@
 #  specific language governing permissions and limitations
 #  under the License.
 
-header:
+ui:
   signIn: 'Sign in'
   darkMode: 'Dark Mode'
   lightMode: 'Light Mode'
-footer:
-  feedback:
-    isEnjoyingTob: 'Enjoying Tour of Beam?'
-    enjoyingLabel: 'Good Experience'
-    notEnjoyingLabel: 'Bad Experience'
   reportIssue: 'Report issue in Jira'
```

Comment:
Everything here is better sorted alphabetically.

Reply:
https://medium.com/flutter-senior/sort-alphabetically-everything-you-can-1e9d7299c4ae

Reply:
Is there a lint for this? If there isn't, we can't enforce this rule on others.

Reply:
No. We may configure some external tools though.
It's not a hard rule also. Just try to sort if you notice. If you don't, it's sad but workable.